tests/chainermn_tests/extensions_tests/test_allreduce_persistent.py

```diff
@@ -39,14 +47,13 @@ def _test(self, comm, model):
 
     def test_allreduce_persistent_cpu(self):
         comm = chainermn.create_communicator('naive')
-        self._test(comm, ExampleModel())
+        model = ExampleModel()
+        self._test(comm, model, False, False)
```

Comment:
Can we have a comment here to make it clear for `chainerx` test and `numpy` test?

Reply:
Fixed.